src/Microsoft.AspNetCore.ResponseCompression/GzipCompressionProvider.cs

```diff
@@ -37,7 +37,13 @@ public bool SupportsFlush
         {
             get
             {
+#if NET461
+                return false;
```

Comment:
Not unless we want to detect the whether it's .NET Framework or .NET Core at runtime. We wanted to avoid this though.